packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -30,17 +30,14 @@ import { CourtesyPage, fiscalCodeRegex, TitleBox } from '@pagopa-pn/pn-commons';
 import PeopleIcon from '@mui/icons-material/People';
 
 import { useAppDispatch, useAppSelector } from '../redux/hooks';
-import {
-  createDelegation,
-  CreateDelegationProps,
-  resetNewDelegation,
-} from '../redux/newDelegation/actions';
+import { createDelegation, resetNewDelegation } from '../redux/newDelegation/actions';
 import { RootState } from '../redux/store';
 import * as routes from '../navigation/routes.const';
 import StyledLink from '../component/StyledLink/StyledLink';
 import DropDownEntiMenuItem from '../component/Deleghe/DropDownEnti';
 import VerificationCodeComponent from '../component/Deleghe/VerificationCodeComponent';
 import ErrorDeleghe from '../component/Deleghe/ErrorDeleghe';
+import { NewDelegationFormProps } from '../redux/delegation/types';
 
 const generateVCode = () =>
```

Comment:
Commentare con TODO tutto ciò che è provvisorio. Immagino che generateVCode sia a fini di test e che andrà eliminato in seguito